R/s3.get.R

```diff
@@ -71,11 +77,19 @@ s3.get <- function (path, bucket.location = "US", verbose = FALSE, debug = FALSE
     last_updated <- strptime(result, format = "%d %b %Y %H:%m:%S", tz = "GMT")
 
     if (last_updated > last_cached) {
-      ans <- fetch()
+      ans <- fetch(path, storage_format, bucket_location, ...)
       s3LRUcache()$set(path, ans)
     } else {
       ans <- s3LRUcache()$get(path)
     }
   }
   ans
 }
+
+load_as_RDS <- function(filename, ...) {
+  readRDS(filename, ...)
+}
+
+load_as_CSV <- function(filename, ...) {
+  read.csv(filename, ...)
```

Comment:
@abelcastilloavant By default I think it should always be FALSE. That was a base library design mistake.